pkg/storage/storagewrappers/combinedtuplereader.go

```diff
@@ -15,16 +16,28 @@ import (
 func NewCombinedTupleReader(
 	ds storage.RelationshipTupleReader,
 	contextualTuples []*openfgav1.TupleKey,
-) storage.RelationshipTupleReader {
-	return &CombinedTupleReader{
+) *CombinedTupleReader {
```

Comment:
FYI so i can make assertions on the inner fields in tests. Also it's a good pattern to return structs instead of interfaces so that the caller can decide which interface it needs. https://medium.com/@cep21/preemptive-interface-anti-pattern-in-go-54c18ac0668a